dataminer/Functions/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Web.md

```diff
@@ -37,7 +37,13 @@ The web component supports two main types of content:
 
 ### Configuring a video thumbnail
 
-You can configure the web component to display a video stream based on an image URL. To do so, configure the URL similar to the value of the *Link* shape data field when you [configure a video thumbnail in Visual Overview](xref:Linking_a_shape_to_a_video_thumbnail).
+You can configure the web component to display a video stream based on an image URL:
```

Comment:
This sentence might be confusing, we don't create a video stream from images. What it can do:

* Display thumbnail images which auto refresh based on an image URL
* Display a video stream based on a video URL